### PR TITLE
Add `textarea` to support testcase output 

### PR DESCRIPTION
**Pull request by Aritra8438**
# Description
This PR adds the textarea to record the output of the testcase.

https://github.com/user-attachments/assets/c7fbcad6-bf58-4a2d-8bd3-7a1c04677de3

# Comments

- [x] Added a textarea
- [x] Added a erase button to erase the output.

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

File: frontend/templates/en.lang [hunk 1]
```diff
@@ -1067,6 +1067,7 @@ problemCreatorDeleteCases = "Delete cases"
 problemCreatorDeleteGroup = "Delete group"
 problemCreatorEditCase = "Edit case"
 problemCreatorEmpty = ""
+problemCreatorEraseOutput = "Erase output"
```
Comment (carlosabcs) [inline frontend/templates/en.lang:1070]: Wouldn't it be better `Clear output` ?

Comment (Aritra8438) [reply frontend/templates/en.lang:1070]: Sure!

**Review by carlosabcs — APPROVED** [review]

**Comment by Aritra8438** [comment]
Please merge 
- [ ] #7702

before this.

**Comment by Aritra8438** [comment]
@heduenas PTAL!

**Pull request merged**